sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -227,7 +235,7 @@ def beautify_dom_info_dict(dom_info_dict, physical_port):
     dom_info_dict['tx2power'] = strip_unit_and_beautify(dom_info_dict['tx2power'], POWER_UNIT)
     dom_info_dict['tx3power'] = strip_unit_and_beautify(dom_info_dict['tx3power'], POWER_UNIT)
     dom_info_dict['tx4power'] = strip_unit_and_beautify(dom_info_dict['tx4power'], POWER_UNIT)
-    if _wrapper_get_sfp_type(physical_port) == 'QSFP_DD':
+    if 'rx5power' in dom_info_dict:
```

Comment:
In case that a QSFP28 transceiver is attached to the QSFP_DD port.

Reply:
even QSFP28 transceiver is attached to the QSFP-DD cage via some adaptor, the type from the EEPROM will not change, right?

Reply:
Yes, because the existing implementation will never update the returned port type, they're constants initialized at the chassis.py